docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/docstring_flags.py

```diff
@@ -19,16 +19,22 @@ def inject_object_flag(
     if isinstance(info, DeprecatedInfo):
         additional_text = f" {info.additional_warn_text}." if info.additional_warn_text else ""
         flag_type = "deprecated"
-        message = f"This API will be removed in version {info.breaking_version}.{additional_text}"
+        messages = [
+            f"This API will be removed in version {info.breaking_version}.",
+            *additional_text.splitlines(),
+        ]
```

Comment:
[1] I think splitting into an array here is unnecessary-- just join the base string to additional text with a newline.